user-journeys/ansible-user-journey-maps.md

```diff
@@ -1,121 +1,97 @@
+```yaml
+Milestone: Start writing playbooks
+  Learn about recommended tools for playbook quality (ansible-lint, handlers, etc)
+  Start validating playbooks
+Milestone: Make playbooks reusable
+  Organize the automation project as it grows.
+  Use roles to structure the automation project.
+  Use Ansible Molecule to improve role quality.
+Milestone: Create execution environment
+Milestone: Execute playbooks remotely
+  Use the AWX REST API.
+  Use the AWX Web UI.
+  Run playbooks on a cron.
+Milestone: Share automation
+  Contribute to a collection.
+  Create a collection.
+  Allow others to run the playbook.
+  Creates AWX job templates.
 ```
 
-### Beginner tasks
-
-- Creates an Ansible development environment
-- Installs Ansible VS Code plugin
-- Installs and/or updates ansible or ansible-core
-- Installs Ansible Lint and developer tooling
-- Installs or updates individual collections
-- Creates and tests playbooks
-- Creates inventories
-- Tests, publishes, and maintains all of these 
-
-### Intermediate tasks
-
-- Improves playbook quality (ansible-lint, handlers, etc)
-- Creates roles
-- Creates job templates (for AWX users)
-- Uses event-driven automation (ansible-rulebook - new)
-- Contributes to collections (bugfixes, modules etc)
-- Tests, publishes, and maintains all of these
-
-### Advanced tasks
-- Manages multiple playbooks, inventories, and jobs with AWX
-- Improves role quality with ansible molecule
-- Creates execution environments with ansible-builder and ansible-navigator
-
-
 ## Operations
 
-## Developer
-
-### Beginner tasks
- - Reviews code of conduct and contributor guidelines for project of interest
- - Sets up development environment with vsCode plugin(s)
- - Fixes issues in existing project, collection, or module
- - Reviews PRs for project of interest
- - Creates module or plugin for local use
-
-### Intermediate tasks
-- Adds new feature to an existing project
-- Develops a dynamic inventory plugin
-- Adds a new module to an existing collection
-- Converts a role into a collection
-- Deprecates a module within a collection
+```yaml
+Milestone: Perform updates in production environments
+  Check the porting guides.
+Milestone: Manage multiple playbooks, inventories, and jobs with AWX
+Milestone: Stand up private Galaxy instance
+```
 
-### Advanced tasks
-- Creates new Ansible project to provide new functionality to the ecosystem
-- Creates collections (reusable plugins, roles, and playbooks)
-- Troubleshoots collections with ansible-navigator
-- Requests a collection be added to the Ansible package
-- Creates execution environments (collections, libraries, and ansible-core version)
-- Tests, publishes, and maintains all of these
+## Developer
 
+```yaml
+Milestone: Learn how to contribute code
+  Review code of conduct and contributor guidelines
+  Review PRs in an Ansible project
+  Fix issues in an Ansible project, collection, or module
+Milestone: Create module or plugin
+Milestone: Add to existing collection
```

Comment:
These two are the same. The only place module/plugins are accepted as a contribution is in a collection.